envvariables.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package helpers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/viper"
+)
+
+// EnvConfig to setup to access to env variables
+type EnvConfig struct {
+	Key          string
+	Variable     string
+	DefaultValue string
+	Required     bool
+}
+
+// GetEnvVariables env variables
```

Comment:
This comment is not golint compliant, must end with a `.`.


```suggestion
// GetEnvVariables extracts configured environment variables and unmarshals them in provided `output` interface.
```